src/core/map.js

```diff
@@ -527,12 +526,15 @@ geo.map = function (arg) {
       m_this.interactor().destroy();
       m_this.interactor(null);
     }
+    m_this.node().off(".geo");
+    $(window).off("resize", resizeSelf);
+    s_exit();
   };
 
   this._init(arg);
 
   // set up drag/drop handling
-  this.node().on("dragover", function (e) {
+  this.node().on("dragover.geo", function (e) {
```

Comment:
This is how jquery namespaces events.  `geo.dragover` would mean event `geo` with namespace `dragover`.  I changed our event names a while back to use [underscores](https://github.com/OpenGeoscience/geojs/blob/master/src/core/event.js#L49) to avoid the confusion.